config.py

```diff
@@ -5,3 +5,5 @@
 axonmyelin_suffix = Path('_seg-axonmyelin.png') # axon + myelin segmentation suffix file name
 axon_suffix = Path('_seg-axon.png')             # axon segmentation suffix file name
 myelin_suffix = Path('_seg-myelin.png')         # myelin segmentation suffix file name
+index_suffix = Path('_IMAGE_index.png')         # image with the index of the axons
```

Comment:
As discussed in the recent meeting, we might want to change the convention of the suffix file names (`index_suffix` and `axonmyelin_index_suffix`) from CAPITAL letters to small letters. Just adding this comment out here, so I don't miss this while reviewing this PR.